plugins/inputs/t128_tank/t128_tank.go

```diff
@@ -0,0 +1,98 @@
+package t128_tank
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const (
+	T128TankAddr = "127.0.0.1"
+	T128TankPort = 11011
+)
+
+var sampleConfig = `
+[[inputs.t128_tank]]
+## A name for the collector which will be used as the measurement name of the produced data.
+# collector_name = ""
+
+## Required. A path for index file.
+# index_file = ""
+
+## Required. Type of topic.
+# topic = "events"
+
+## Required. Port Number to get tank data from.
+# port_number = 11011
+
+## Required. Port Address to get tank data from.
+# port_address = "127.0.0.1"
+`
+
+type T128Tank struct {
+	CollectorName string `toml:"collector_name"`
```

Comment:
Is `CollectorName` used? It used to be used for creating a log file. Is it just the measurement name now?

Reply:
just a measurement name. I will remove it since we can always rely on name override